### PR TITLE
[10.0][MIGR] account_invoice_force_number

### PR DESCRIPTION
**Pull request by tafaRU**
~~This PR is still WIP according to its label.~~

### CODE REVIEW

File: account_invoice_force_number/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Force Invoice Number
+====================
+
+This module allows to force the invoice numbering. It displays the internal_number field. If user fills that field, the typed value will be used as invoice (and move) number. Otherwise, the next sequence number will be retrieved and saved. So, the new field has to be used when user doesn't want to use the default invoice numbering for a specific invoice.
```
Comment (eLBati) [inline account_invoice_force_number/README.rst]: ```internal_number``` -> ```move_name```

**Review by eLBati — CHANGES_REQUESTED** [review]
@tafaRU thanks, just a remark

**Review by eLBati — APPROVED** [review]
thanks

**Comment by tafaRU** [comment]
@eLBati, what about errors on https://travis-ci.org/OCA/account-invoicing/jobs/219682034?
Do you think they are related?

**Comment by eLBati** [comment]
@tafaRU uhm I don't know, they need to be checked

**Review by eLBati — CHANGES_REQUESTED** [review]
travis check

**Comment by tafaRU** [comment]
@eLBati, I just tried to restart that build was red and now, I don't know why, it appears as green!

**Comment by eLBati** [comment]
@tafaRU thanks, I would squash some commits

**Comment by tafaRU** [comment]
@eLBati, I squashed commits a bit!

**Review by eLBati — APPROVED** [review]
thanks

**Comment by stevech091** [comment]
I have tried it and it works

**Comment by LoisRForgeFlow** [comment]
Runbot not working, rebuild and still not able to log in.

**Comment by tafaRU** [comment]
@lreficent, amended last commit to trigger all builds, please try again!

**Review by LoisRForgeFlow — APPROVED** [review]
Technical and functional review, LGTM :+1:

**Pull request merged**